fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biomes/BiomeVariant.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.biomes;
+
+import net.minecraft.world.biome.Biome;
+
+/**
+ * Represents a biome variant and its corresponding chance
+ */
+public class BiomeVariant {
+
+	private final Biome variant;
+	private final double chance;
+
+	/**
+	 * @param variant the variant biome
+	 * @param chance the chance of replacement of the biome into the variant
```

Comment:
```suggestion
	 * @param The chance at which the variant will replace a bound parent
```

Reply:
`chance` must be right after `@param` as that's required by javadocs.

Reply:
Oops, I mistyped, `chance` should still be there